test/regressions/index.test.js

```diff
@@ -119,7 +119,7 @@ async function main() {
           window.muiTogglePickerMode();
         });
         await takeScreenshot({
-          testcase: await page.waitForSelector('[role="dialog"]'),
+          testcase,
```

Comment:
Since the state is reset, the dialog is no longer open.

Reply:
Should we apply the actions back in the end-to-end test? Otherwise, we probably can remove `/regression-pickers/UncontrolledDateTimePicker-mobile`.

Reply:
Why would we remove the test? The behavior changed. The test is still relevant.

Reply:
The test is still relevant because it still screenshots the input  which is different between mobile and desktop.

Reply:
> The test is still relevant because it still screenshots the input which is different between mobile and desktop.

Ok, I didn't realize the input was different, to me, the previous snapshot were clearer

**desktop**
![b7b169fc1d46ca6a0080c046d12f755e](https://user-images.githubusercontent.com/3165635/117433803-2fc82600-af2c-11eb-83bc-a59c07e75e58.png)

**mobile**
![a4536d7f28c5ce643c033979b9b54b9f](https://user-images.githubusercontent.com/3165635/117433809-30f95300-af2c-11eb-8645-e29e3e7db42e.png)